examples/fastify/docker/docker-compose.yaml

```diff
@@ -4,13 +4,14 @@ services:
   collector:
     image: otel/opentelemetry-collector:0.38.0
 #    image: otel/opentelemetry-collector:latest
-    command: ["--config=/conf/collector-config.yaml", "--log-level=DEBUG"]
+    command: ["--config=/conf/collector-config.yaml"]
     volumes:
       - ./collector-config.yaml:/conf/collector-config.yaml
     ports:
       - "9464:9464"
       - "4317:4317"
       - "4318:4318"
+      - "55681:55681"
```

Comment:
thanks for adding this fix.
The real issue is that this fastify example is using this `@opentelemetry/exporter-collector` very old package, which still uses the old port 55681 instead of the new one which is 4317/4318.

I think the proper fix would be to upgrade the dependencies to use the otlp exporter package (which by default uses the new port). This is not a blocker for the PR, only a suggestion if you have time and motivation :)

Reply:
@blumamir Thank you, I wasn't aware of this. I will sort it ou